src/compute-types/src/dataflows.rs

```diff
@@ -75,37 +74,28 @@ pub struct DataflowDescription<P, S: 'static = (), T = mz_repr::Timestamp> {
 }
 
 impl<T> DataflowDescription<Plan<T>, (), mz_repr::Timestamp> {
-    /// Tests if the dataflow refers to a single timestamp, namely
-    /// that `as_of` has a single coordinate and that the `until`
-    /// value corresponds to the `as_of` value plus one, or `as_of`
-    /// is the maximum timestamp and is thus single.
-    pub fn is_single_time(&self) -> bool {
-        // TODO: this would be much easier to check if `until` was a strict lower bound,
-        // and we would be testing that `until == as_of`.
```

Comment:
Removing this TODO as, while the statement made is correct, changing the semantics of `until` like this is not trivial and doing it just to simplify the logic of this one method seems misguided. The current check is not that bad at all.